lib/Server.js

```diff
@@ -389,6 +389,23 @@ function Server(compiler, options) {
 			const pems = selfsigned.generate(attrs, {
 				algorithm: "sha256",
 				days: 30,
+				extensions: [{
+					name: "subjectAltName",
+					altNames: [
+						{
+							type: 2, // DNS
+							value: "localhost"
+						},
+						{
+							type: 6, // URI
```

Comment:
Could we have some explanation as to which browsers look to each of these altName fields in order to function w/o warnings?

Reply:
I don't know of any browser that requires it; when I tested w/ Chrome, it wasn't required.  I only included it based on [this comment](https://github.com/angular/angular-cli/issues/4839#issuecomment-314901315).